lua/nvim-treesitter/query.lua

```diff
@@ -192,6 +192,36 @@ function M.get_capture_matches(bufnr, capture_string, query_group)
     return matches
 end
 
+function M.find_best_match(bufnr, capture_string, query_group, filter_predicate, scoring_function)
+    if not string.sub(capture_string, 1,2) == '@' then
+      print('capture_string must start with "@"')
```

Comment:
Meh, why printing here if it's an error ? `nvim_err_writln` should be used.

Reply:
We always you print in this repo. But I fixed it.